product_docs/docs/transporter/2_preview/installing/installing_debian.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: "Installation for Debian and derivatives"
+---
+
+<div id="installation_for_debian" class="registered_link"></div>
+
+## Prerequisites/repository setup
+
+Quick setup:
+
+```shell
+curl -1sLf \
+  'https://downloads.enterprisedb.com/j1eFdSoy7eEBo9pK/dev/setup.deb.sh' \
+  | sudo -E bash
+```
+
+Set up a specific distribution or architecture:
+
+```shell
+curl -1sLf \
+  'https://downloads.enterprisedb.com/j1eFdSoy7eEBo9pK/dev/setup.deb.sh' \
+  | sudo -E distro=some-distro codename=some-codename arch=some-arch bash
+```
+
+Install manually:
+
+```shell
+distro="some-distro" # for example "ubuntu"
+codename="some-codename" # for example "xenial"
+
+apt-get install -y debian-keyring  # debian only
+apt-get install -y debian-archive-keyring  # debian only
+apt-get install -y apt-transport-https
+# For Debian Stretch, Ubuntu 16.04 and later
+keyring_location=/usr/share/keyrings/enterprisedb-dev-archive-keyring.gpg
+# For Debian Jessie, Ubuntu 15.10 and earlier
+keyring_location=/etc/apt/trusted.gpg.d/enterprisedb-dev.gpg
+curl -1sLf 'https://downloads.enterprisedb.com/j1eFdSoy7eEBo9pK/dev/gpg.8FE52FF870688836.key' |  gpg --dearmor >> ${keyring_location}
+curl -1sLf 'https://downloads.enterprisedb.com/j1eFdSoy7eEBo9pK/dev/config.deb.txt?distro=${distro}&codename=${codename}' > /etc/apt/sources.list.d/enterprisedb-dev.list
```

Comment:
Wasn't sure if this was curl nomenclature so I didn't change these.

Reply:
This is OK like it's now. $distro and $codename are vars set on lines 28-29